src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1208,13 +1214,16 @@ end
     unique!(df::AbstractDataFrame)
     unique!(df::AbstractDataFrame, cols)
 
-Delete duplicate rows of data frame `df`, keeping only the first occurrence of unique rows.
-When `cols` is specified, the returned `DataFrame` contains complete rows,
-retaining in each case the first instance for which `df[cols]` is unique.
-`cols` can be any column selector ($COLUMNINDEX_STR; $MULTICOLUMNINDEX_STR).
+Delete duplicate rows of data frame `df`, keeping only the first occurrence of
+unique rows. When `cols` is specified, the returned `DataFrame` contains
```

Comment:
BTW, "delete" seems to imply in-place operation. Maybe this, inspired by `filter`?
```suggestion
Return a data frame containing only the first occurrence of unique rows in `df`.
When `cols` is specified, the returned `DataFrame` contains
```